unfoldingWord/translationCore
# USFM Import TL Bible Fix

#### This pull request addresses:
There was a bug in the project validation stepper that skipped generating a target language for usfm imports

#### How to test this pull request:
Test using usfm projects and ensure the target language bible is created, and checks work as expected.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2572)
